test/libsolidity/semanticTests/array/cleanup_during_multi_element_per_slot_copy.sol

```diff
@@ -0,0 +1,22 @@
+contract C {
+  uint32[] s;
+  constructor()
+  {
+    s.push();
+    s.push();
+  }
+  function f() external returns (uint)
+  {
+    (s[1], s) = (4, [0]);
+    s = [0];
+    s.push();
+    return s[1];
+    // used to return 4 via IR.
```

Comment:
Ah, it's a test case for #12600? Then maybe this could have a comment mentioning this case?

Reply:
I still think this could use a comment. Before I read https://github.com/ethereum/solidity/issues/12600#issuecomment-1150951006, it was not clear to me what's happening here.